src/SourceLocator/SourceStubber/PhpStormStubsMap.php

```diff
@@ -0,0 +1,4910 @@
+<?php
+
+declare(strict_types=1);
+
+$files = [
```

Comment:
Would it be OK to make this a static class with a constant? I don't like having global variables declared under an `src/` path, so this entire structure should be the result of a `var_export()`, and pushed into a `private` class constant

Reply:
The reasoning being that I'd rather go through the composer autoloader here, rather than dynamic paths

Reply:
Done.